dashboard/app/reporting.go

```diff
@@ -1281,61 +1273,6 @@ func (bug *Bug) updateReportings(cfg *Config, now time.Time) error {
 	return nil
 }
 
-func queryCrashesForBug(c context.Context, bugKey *db.Key, limit int) (
-	[]*Crash, []*db.Key, error) {
-	var crashes []*Crash
-	keys, err := db.NewQuery("Crash").
-		Ancestor(bugKey).
-		Order("-ReportLen").
-		Order("-Time").
-		Limit(limit).
-		GetAll(c, &crashes)
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to fetch crashes: %v", err)
-	}
-	return crashes, keys, nil
-}
-
-func queryJobsForBug(c context.Context, bugKey *db.Key, jobType JobType) (
-	[]*Job, []*db.Key, error) {
-	var jobs []*Job
-	keys, err := db.NewQuery("Job").
-		Ancestor(bugKey).
-		Filter("Type=", jobType).
-		Filter("Finished>", time.Time{}).
-		Order("-Finished").
-		GetAll(c, &jobs)
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to fetch fix bisections: %v", err)
-	}
-	return jobs, keys, nil
-}
-
-func queryCrashForJob(c context.Context, job *Job, bugKey *db.Key) (*Crash, error) {
-	// If there was no crash corresponding to the Job, return.
-	if job.CrashTitle == "" {
-		return nil, nil
-	}
-	// First, fetch the crash who's repro was used to start the bisection
-	// job.
-	crash := new(Crash)
-	crashKey := db.NewKey(c, "Crash", "", job.CrashID, bugKey)
-	if err := db.Get(c, crashKey, crash); err != nil {
-		return nil, err
-	}
-	// Now, create a crash object with the crash details from the job.
-	ret := &Crash{
```

Comment:
TODO: probably we need to preserve this behavior.

Reply:
Looked at the code once more -- no, I think we don't need to preserve it. Everywhere we only take C/syz repro links from the crash.

https://github.com/google/syzkaller/blob/35d9ecc508aef508b67ee7986a7abb0864e74f8e/dashboard/app/jobs.go#L1534-L1538

https://github.com/google/syzkaller/blob/35d9ecc508aef508b67ee7986a7abb0864e74f8e/dashboard/app/templates.html#L362-L364